reqs/management/commands/import_reqs.py

```diff
@@ -219,12 +219,32 @@ def __init__(self, keywords=None):
         self.keywords = KeywordProcessor(keywords)
         self.connections = []
         self.req_ids = set()
+        self.last_id = "0.0"
+
+    def fix_excel_decimals(self, reqid):
+        """
+        Excel strips trailing zeroes, so for example what should be 1.20 is 1.2
+        in the data.
+        Since the rows are sequential (or should be!), we should be able to
+        account for this.
+        """
+        policy, requirement = reqid.split(".")
+        last_policy, last_requirement = self.last_id.split(".")
+        if policy == last_policy:
+            if last_requirement.endswith("9"):
+                sequential = str(int(last_requirement) + 1)
+                if sequential[:len(requirement)] == requirement:
```

Comment:
This feels risky due to dropping data. What about something like:
```python
if sequential.rstrip('0') == requirement:
```
The situation I'm imagining is something like `1.234569` followed by `1.23`, which shouldn't come up, but I don't think we need to risk it.